int/api/websites/all.go

```diff
@@ -53,48 +59,91 @@ smart contract Massa Station is connected to. Once this data has been fetched fr
 the various website storer contracts, the function builds an array of Registry objects
 and returns it to the frontend for display on the Registry page.
 */
+// Registry fetches the registry data for the given AppConfig.
 func Registry(config *config.AppConfig) ([]*models.Registry, error) {
 	client := node.NewClient(config.NodeURL)
 
-	// Fetch website names to display
 	websiteNames, err := filterEntriesToDisplay(*config, client)
 	if err != nil {
-		return nil, fmt.Errorf("filtering keys to be displayed at '%s': %w", config.DNSAddress, err)
+		return nil, fmt.Errorf("failed to filter keys to be displayed at '%s': %w", config.DNSAddress, err)
 	}
 
-	// Fetch DNS values of the website names: contractAddress, Description.
 	dnsValues, err := node.ContractDatastoreEntries(client, config.DNSAddress, websiteNames)
 	if err != nil {
-		return nil, fmt.Errorf("reading keys '%s' at '%s': %w", websiteNames, config.DNSAddress, err)
+		return nil, fmt.Errorf("failed to read keys '%s' at '%s': %w", websiteNames, config.DNSAddress, err)
 	}
 
-	// Pre-allocate registry with the estimated capacity
-	registry := make([]*models.Registry, 0, len(dnsValues))
+	registry, err := processDNSValues(client, dnsValues, websiteNames)
+	if err != nil {
+		return nil, err
+	}
+
+	sortRegistry(registry)
```

Comment:
> are you sure this function does something? i would expect to have the sorted registry returned: `registry = sortRegistry(registry)`

Since sort.Slice modifies the slice in place, there's no need to return it explicitly.

Reply:
yes i am 100% sure  👯